### ISSUE TITLE
Question: Training With 2 classes error

### ISSUE THREAD

**Issue by AndLydakis** [DrSleep/light-weight-refinenet#52]
Hello, I am trying to train a classifier for a single object, so classes are 'object' and 'no_object', but I am getting the following error in line 280 in train.py when loss.backward() is called:
`/pytorch/aten/src/THCUNN/SpatialClassNLLCriterion.cu:104: void cunn_SpatialClassNLLCriterion_updateOutput_kernel(T *, T *, T *, long *, T *, int, int, int, int, int, long) [with T = float, AccumT = float]: block: [0,0,0], thread: [479,0,0] Assertion `t >= 0 && t < n_classes` failed.`

My images and labels are in the NYU dataset format, and the pass the initial checks. For parameters I have only changed the `NUM_CLASSES` parameter in `config.py` to `[2] * 3`. Is that the correct way or am I missing something ?

I'm running ubuntu 18.0 with Cuda 10, CUDNN 7.
If you need any additional details let me know.

**Comment by AndLydakis** [comment]
It turned out to be an issue on how the labels are parsed, more of a PIL issue than the network implementation, closing.
